diff --git handlers/views_handler_field.inc handlers/views_handler_field.inc
index d1dabe8..4036dff 100644
--- handlers/views_handler_field.inc
+++ handlers/views_handler_field.inc
@@ -258,6 +258,16 @@ class views_handler_field extends views_handler {
           'edit-options-alter-make-link' => array(1)
         ),
       );
+      $form['alter']['rel'] = array(
+        '#title' => t('Rel Text'),
+        '#type' => 'textfield',
+        '#default_value' => $this->options['alter']['rel'],
+        '#description' => t('Include Rel attribute for use in lightbox2 or other javascript utility.'),
+        '#process' => array('views_process_dependency'),
+        '#dependency' => array(
+          'edit-options-alter-make-link' => array(1)
+        ),
+      );
       $form['alter']['prefix'] = array(
         '#title' => t('Prefix text'),
         '#type' => 'textfield',
@@ -609,6 +619,11 @@ class views_handler_field extends views_handler {
       $options['attributes']['class'] = $class;
     }
 
+    $rel = strtr($alter['rel'], $tokens);
+    if ($rel) {
+      $options['attributes']['rel'] = $rel;
+    }
+
     $target = check_plain(trim(strtr($alter['target'],$tokens)));
     if (!empty($target)) {
       $options['attributes']['target'] = $target;
